docker/lib/dependabot/docker/file_updater.rb

```diff
@@ -183,6 +198,11 @@ def new_yaml_image(file)
         "#{prefix}#{dependency.name}#{tag}#{digest}"
       end
 
+      def new_yaml_tag(file)
+        elt = dependency.requirements.find { |r| r[:file] == file.name }
+        elt.fetch(:source)[:tag] || ""
```

Comment:
Maybe I'm just being blind, but what does `elt` stand for? I see you copied from the method above so not looking for it to be changed, just to understand...

Reply:
`elt` == `element` changed it to be explicit.
(it's a hold-over from javascript days of saying `elt = $.(#foo)` :)

Reply:
Ah, thanks! I don't come from a javascript background, so I appreciate the historical context.